app/jobs/dunning_campaigns/bulk_process_job.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module DunningCampaigns
+  class BulkProcessJob < ApplicationJob
```

Comment:
is this job going to be called from another location?

just wondering if it's not simpler to directly call `DunningCampaigns::BulkProcessService.call` from the clock job

Reply:
as running this service will take longer with a growing number of organizations and customers, IMO it is better to free clock worker and just queue this job an let the normal sidekiq workers take care of it. We can handle queue priority, and more stuff that is not available with clock.

Reply:
fair enough